scripts/release/issue-list.sh

```diff
@@ -23,6 +23,15 @@ function usage()
     exit 1
 }
 
+function discard_content_with_empty_issues() {
+  echo $1 | grep -q '[0-9]'
+  if [[ "$?" != 0 ]]; then
```

Comment:
> features="\n# Features\n"

That will add an unnecessary line at the beginning if features has more than 3 lines.